src/toast/util/toastTypes.ts

```diff
@@ -10,6 +10,7 @@ export type ToastAction =
   | {
       type: "DISPLAY";
       toastData: Omit<ToastData, "id"> & {id: string};
+      limit?: number;
```

Comment:
We shouldn't need to provide this DISPLAY action with the limit. Let's find another solution which doesn't require passing the limit to the action.